uboot.env.in

```diff
@@ -5,8 +5,8 @@ boot_extlinux=sysboot ${devtype} ${devnum}:${bootpart} any ${scriptaddr} ${prefi
 boot_prefixes=/ /boot/
 boot_script_dhcp=boot.scr.uimg
 boot_scripts=boot.scr.uimg boot.scr
-boot_targets=mmc0 usb0 pxe dhcp 
-bootcmd=run distro_bootcmd
+boot_targets=mmc0 usb0 pxe dhcp
+bootcmd=env exists initrd_high || env default initrd_high; env exists fdt_high || env default fdt_high; run distro_bootcmd
```

Comment:
Ah, so the `env default` did work! This does *look* a bit more hacky than setting the env variables directly, but I guess this is saner to rely on defaults from u-boot instead of setting yet another constant in the gadget code.